File: python/src/tests/t_event_locks_max.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

maxindex = atrshmlog.get_event_locks_max()

print('event locks max : ' + str(maxindex) + ' : ')

newindex  = int(sys.argv[1])

print('event locks max : ' + str(newindex) + ' : ')

oldindex = atrshmlog.set_event_locks_max(newindex)

print('event locks max : ' + str(oldindex) + ' : ')

maxindex = atrshmlog.get_event_locks_max()

print('event locks max : ' + str(maxindex) + ' : ')

result = atrshmlog.attach()

oldindex = atrshmlog.set_event_locks_max(newindex + 100)

print('event locks max : ' + str(oldindex) + ' : ')

maxindex = atrshmlog.get_event_locks_max()

print('event locks max : ' + str(maxindex) + ' : ')

print (' ')

exit(0)

# end of test

File: perl/src/tests/t_event_locks_max.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$maxindex = Atrshmlog::get_event_locks_max();

print "event locks max : $maxindex : \n";

$newindex  = shift @ARGV;

print "event locks max : $newindex : \n";

$oldindex = Atrshmlog::set_event_locks_max($newindex);

print "event locks max : $oldindex : \n";

$maxindex = Atrshmlog::get_event_locks_max();

print "event locks max : $maxindex : \n";

$result = Atrshmlog::attach();

$oldindex = Atrshmlog::set_event_locks_max($newindex + 100);

print "event locks max : $oldindex : \n";

$maxindex = Atrshmlog::get_event_locks_max();

print "event locks max : $maxindex : \n";


print " \n";

exit (0);

# end of main
